### PR TITLE
variable env

### PR DESCRIPTION
**Pull request by simonwallin**
Ability to set red bee environment. Useful when developing and testing against red bee's stage env.

### CODE REVIEW

**Comment by simonwallin** [comment]
Old PR, but still relevant. Please have a look and give some feedback.

File: src/api/exposure.js [hunk 1]
```diff
@@ -1,20 +1,33 @@
 //const debug = require("debug")("mgmt");
 
-const EXPOSURE_API_ENDPOINT = "https://exposureapi.emp.ebsd.ericsson.net/v2";
-
+const EXPOSURE_API_ENDPOINT_PROD =
+  "https://exposureapi.emp.ebsd.ericsson.net/v2";
+const EXPOSURE_API_ENDPOINT_STAGE =
+  "https://psempexposureapi.ebsd.ericsson.net/v2";
```
Comment (slowmove) [inline src/api/exposure.js:6]: I don't know if we should have `v2` as a part of these urls even though all endpoint as of right now might have it 🤔

Comment (simonwallin) [reply src/api/exposure.js:6]: Agree. I did not want to change it myself though, since I don't really understand why it is that way.

Comment (slowmove) [reply src/api/exposure.js:6]: Ah, I'll look into it then

**Review by slowmove — APPROVED** [review]

**Pull request merged**